04-simulate-an-election/election.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+
+def elect_in_region(election_result, candidate_chances):
+    if election_result <= candidate_chances:
+        return 'a'
+    else:
+        return 'b'
+
+def simulate_a_single_election():
+    candidate_a_chances = {
+        "region1": 87,
+        "region2": 65,
+        "region3": 17
+    }
+
+    candidate_a_elected_count = 0
+    candidate_b_elected_count = 0
+
+    for region in candidate_a_chances:
+        elected = elect_in_region(election_result=int(random() * 100), candidate_chances=candidate_a_chances[region])
+
+        if elected == "a":
+            candidate_a_elected_count += 1
+        else:
+            candidate_b_elected_count += 1
+
+    if candidate_a_elected_count > candidate_b_elected_count:
+        return 'a'
+    else:
+        return 'b'
+
+def simulate_elections(elections_count):
+    candidate_a_elections_count = 0
+    candidate_b_elections_count = 0
+
+    for i in range(0, elections_count):
+        election_result = simulate_a_single_election()
+
+        if election_result == 'a':
+            candidate_a_elections_count += 1
+        else:
+            candidate_b_elections_count += 1
+
+    print("Candidate A won", candidate_a_elections_count, "times")
+
+    print("Candidate B won", candidate_b_elections_count, "times")
+
+    results = {
+        'count': elections_count,
```

Comment:
Please change the key of the first value to `elections_count`.
Note that you don't really have to specify the elections count here, as you already know it when calling this function, and can specify it from the `main` to any following function you require.